### DISCUSSION TITLE
Best practice for Clickhouse integration?

### DISCUSSION THREAD

**Discussion by clayheaton** [dagster-io/dagster#20085] [Q&A] [answered]
I'm just starting a setup of Clickhouse (as a primary Superset backend) and want to use my existing Dagster setup to manage the data in Clickhouse. 

Is there an official Dagster integration with Clickhouse? Or is it recommended to use the [Clickhouse Connect](https://clickhouse.com/docs/en/integrations/python) library?

**Comment by garethbrickman** [comment, marked as answer]
There is no official integration for Clickhouse at this time, however it has been feature requested in #5465

**Reply by clayheaton** [reply]
Ok, thanks!
